### ISSUE TITLE
python build regression in 1.0 release

### ISSUE THREAD

**Issue by anthraxx** [radare/radare2-bindings#142]
building with the radare2 1.0 release installed via:
```
./configure --prefix=/usr --enable=ctypes --enable=python
make
```

results in a regression:

```
r_core_wrap.cxx: In function ‘int RCore_project_open(RCore*, char*)’:
r_core_wrap.cxx:7430:40: error: too few arguments to function ‘int r_core_project_open(RCore*, const char*, bool)’
   return r_core_project_open(self, file);
                                        ^
In file included from r_core_wrap.cxx:3309:0:
/usr/include/libr/r_core.h:418:11: note: declared here
 R_API int r_core_project_open(RCore *core, const char *file, bool thready);
           ^~~~~~~~~~~~~~~~~~~
```

PS: even through it raises an error, the ```make``` call exists with 0, maybe this should be fixed too?

**Comment by radare** [comment]
Fixed in 1.0.1

**Comment by anthraxx** [comment]
@radare thx, confirming that it works
